scripts/lib/install

```diff
@@ -368,7 +368,7 @@ fi
 
 if [ -n "$USE_CERTBOT" ]; then
     "$ZULIP_PATH"/scripts/setup/setup-certbot \
-        --no-zulip-conf --method=standalone \
+        --method=standalone \
```

Comment:
This runs before `puppet`, so I think our deploy hook won't be in place yet.  But I guess that's OK, since we won't have installed nginx yet either?  Might be worth a comment on this correctness detail.

Reply:
We install nginx at the same time we install the hook, yeah.  I don't entirely think that's worth a comment in the code, but I can add one if it'd have been helpful to you.